canonical/ubuntu.com
# /download/server 24.04 updates

## Done

- Updated copy as per [doc](https://docs.google.com/document/d/1Qob4Ni_y1ffIdSWqaw2xH0dxiHZQmkRGP5nuoBhIfyM/edit)
- Drive bys:
  - Updated previous lts versions on releases.yaml and added server iso sizes 

## QA

- View the site locally in your web browser at: http://0.0.0.0:8001/download/server
    - Be sure to test on mobile, tablet and desktop screen sizes
- See that the requested changes have been made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10052
